lib/shopify_cli/resources/env_file.rb

```diff
@@ -53,7 +61,7 @@ def parse_external_env(directory = Dir.pwd)
       end
 
       property :api_key, required: true
-      property :secret, required: true
+      property :secret
```

Comment:
Need to get rid of this requirement to support environment configs with just API key and registration ID.